tools/ssh-bastion-ec2/ssh-bastion-ec2

```diff
@@ -117,12 +119,12 @@ def ssh_ec2(remote_host, bastion_ip, key):
   retVal = 1
   try:
     tmp_key_path, ssh_cmd = _get_ssh_command(remote_host, bastion_ip, key)
-    # print("SSH_COMMAND: ", ssh_cmd)
-    retVal = os.system(ssh_cmd)
+    logging.info("SSH_COMMAND: %s", ' '.join(ssh_cmd))
```

Comment:
Should this be `logger`? There is more below.

Reply:
Done.